qual_id/categories/anime.py

```diff
@@ -0,0 +1,7 @@
+from ..category import Category
```

Comment:
Please can you remove this category file and add it (along with tests etc.) in a separate PR?